user-manual/transfer/transfer.rst

```diff
@@ -147,6 +147,13 @@ standard transfer. For more information on how standard transfers can be
 structured, see :ref:`basic transfers <basic-transfers>` and subsequent
 sections below.
 
+**Zipped directory**: Zipped directory transfers are a variant of the default
+transfer type in Archivematica. All materials can be transferred using the
+Zipped directory transfer type; the material just happens to be in a ``.zip``,
+``.tgz``, or ``.tar.gz`` at the outset. When a Zipped directory transfer
+starts the zip will be decompressed and the material then processed using the
+Standard transfer workflow documented above.
```

Comment:
Maybe worth a line here explaining that the contents of the zip/tgz/tar.gz should match the structure of the standard transfer?

Reply:
Thanks Alex. I provided a bit more exposition. Hopefully that works. I'll submit this for formal review from the team, but feel free to add more comments. too.